servers/rendering/renderer_rd/renderer_compositor_rd.cpp

```diff
@@ -46,6 +46,8 @@ void RendererCompositorRD::blit_render_targets_to_screen(DisplayServer::WindowID
 		RID rd_texture = storage->texture_get_rd_texture(texture);
 		ERR_CONTINUE(rd_texture.is_null());
 
+		// TODO if keep_3d_linear was set when rendering to this render target we need to add a linear->sRGB conversion in.
+
```

Comment:
will do this separately and maybe move some of the settings into specialization constants.